test/metabase/api/macros/defendpoint/open_api_test.clj

```diff
@@ -51,8 +43,8 @@
   (testing "nested data structures are still fixed up"
```

Comment:
maybe this test should actually be removed since it doesn't actually need to be "fixed up"?